pipelines/controller/src/main/java/org/openmrs/analytics/HiveTableManager.java

```diff
@@ -86,6 +86,17 @@ private void createResourceTable(
               thriftServerParquetPath,
               resource);
       statement.execute(sql);
+
+      // Drop canonical table if exists.
+      sql = String.format("DROP TABLE IF EXISTS default.%s", resource);
+      statement.execute(sql);
+
+      // Create canonical table with latest parquet files.
+      sql =
+          String.format(
+              "CREATE TABLE IF NOT EXISTS default.%s USING PARQUET LOCATION '%s/%s/%s'",
+              resource, THRIFT_CONTAINER_PARQUET_PATH_PREFIX, thriftServerParquetPath, resource);
```

Comment:
nit: Have you done `mvn clean package` on the latest version of your PR? Asking because I think it will change the formatting here.

Reply:
yes. I ran again and I find no difference.